docs/zh-cn/manual/intro-to-pipeline.md

```diff
@@ -59,7 +59,19 @@ $ pipcook run /path/to/your/pipeline-config.json
    ┗ 📜index.js
 ```
 
-通过下面的脚本即可使用你训练的脚本了：
+为了使用训练好的模型，还需要完成以下步骤：
+
+```sh
+$ npm install
+```
+
+上述的命令会帮助你安装依赖，它们包括插件以及插件所依赖的 Python 包。Pipcook 也提供了一条命令，来使用 [tuna](https://mirrors.tuna.tsinghua.edu.cn/) 镜像来下载 Python 包：
+
+```sh
+$ BOA_TUNA=1 npm install
```

Comment:
Is this env variable used in script below?

Reply:
This is used by boa's installation scripts which are introduced by at #280.